open_alchemy/build/__init__.py

```diff
@@ -246,3 +249,89 @@ def execute(*, spec: typing.Any, name: str, path: str) -> None:
         spec=spec_str,
         init=init,
     )
+
+    # Build a distribution package if necessary.
+    if format_:
+        pkg_dir = pathlib.Path(path) / name
+        build_dist(str(pkg_dir), format_)
+
+
+def build_dist(path: str, format_: str) -> None:
+    """
+    Build a distibution package.
+
+    Args:
+        path: The package directory.
+        format: The package format to build ("sdist", "wheel" or "all").
+    """
+    formats = {"sdist", "wheel", "all"}
+    fmt = format_.lower()
+    if fmt not in formats:
+        raise ValueError(
```

Comment:
You can avoid this with `Enum`. Also, all exceptions that are raised should inherit from the `BaseError` in `exceptions.py` and be purpose driven. For example, there is already a `BuildError`, do you think that is appropriate here?